<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineVoltaje">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Display Semib</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Neuronas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltaje:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Posición X:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineY">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Posición Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineBlue">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRed">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineGreen">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVerificar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>761</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Archivos</string>
     </attribute>
     <widget class="QPushButton" name="buttonClean">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>80</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpiar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRead">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Importar archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSave">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar archivo</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textNeuronas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>451</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAgregarFinal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Neurona al Final</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAgregar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Neurona al Inicio</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Buscar</string>
     </attribute>
     <widget class="QTextEdit" name="textBuscar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>341</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBuscar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar Neurona:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineBuscar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tabla</string>
     </attribute>
     <widget class="QPushButton" name="buttonSaveTable">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar tabla</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAdd">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>120</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar a Tabla</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableNeuronas">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>701</width>
        <height>221</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="buttonImportar">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Importar a tabla</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Dibujar</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsViewNeuronas">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>11</y>
        <width>591</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Dibujar Neurona</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>60</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Borrar vista</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuADNMINISTRADOR_DE_NEURONAS">
    <property name="title">
     <string>ADNMINISTRADOR DE NEURONAS</string>
    </property>
   </widget>
   <addaction name="menuADNMINISTRADOR_DE_NEURONAS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
